alkira/resource_alkira_billing_tag.go

```diff
@@ -11,17 +10,22 @@ import (
 func resourceAlkiraBillingTag() *schema.Resource {
 	return &schema.Resource{
 		Description: "Manage Billing Tag.",
-		Create: resourceBillingTag,
-		Read:   resourceBillingTagRead,
-		Update: resourceBillingTagUpdate,
-		Delete: resourceBillingTagDelete,
+		Create:      resourceBillingTag,
+		Read:        resourceBillingTagRead,
+		Update:      resourceBillingTagUpdate,
+		Delete:      resourceBillingTagDelete,
 
 		Schema: map[string]*schema.Schema{
 			"name": {
 				Description: "Billing Tag Name.",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"description": {
+				Description: "Billing Tag Description.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
 			"tag_id": {
 				Description: "Billing Tag ID.",
 				Type:        schema.TypeInt,
```

Comment:
Should we keep this one as TypeInt? seems like we are only using it as a string

Reply:
I should remove this one now... No need anymore. The default Id should just work..